pkg/proxy/proxy.go

```diff
@@ -120,13 +94,13 @@ func (k *Currency) OnRequest(id uint64, payload []byte, hasSURB bool) ([]byte, e
 	k.log.Debugf("Handling request %d", id)
 
 	// Send request to HTTP RPC.
-	req, err := common.RequestFromJson(k.ticker, k.chaindID, payload)
+	req, err := common.RequestFromJson(k.ticker, payload)
```

Comment:
RequestFromJson still has the old signature.

`RequestFromJson(expectedTicker string, expectedChainId int, rawRequest []byte) (*CurrencyRequest, error)`

Reply:
`RequestFromJson(expectedTicker string, expectedChainId int, rawRequest []byte) (*CurrencyRequest, error)`

needs to change to:
`RequestFromJson(expectedTicker string, rawRequest []byte) (*CurrencyRequest, error)`